### ISSUE TITLE
Date range picker only show once when using customize input

### ISSUE THREAD

**Issue by MrShayne** [nathanreyes/v-calendar#302]
Hi I am using Vuetify as my framework.
  I use the Vuetify input component as a customize input for v-date-picker, I do everything like the official document says -- v-bind='inputProps', v-on='inputEvents'. And I set 'popover.visibility' to 'focus'. But popover will only show at the first time I click on the input element, after that, even the input element is focused the popover won't show. 

  This only happened in Vuetify input component though, I tried HTML input element and it worked fine.

  Would appreciate your help

**Comment by CImrie** [comment]
I'm having the same problem. It looks like when using a slot, the `PopoverRef` component depends on `this.$slots.default[0].elm` being the source of any emitted `blur/focusout` event. 
The `blur` event can only be fired if the element has been focused on directly. `div` elements are not focusable by default, and so the `blur` event doesn't fire (at least this is my understanding!).

I managed to resolve this by catching my `input` blur event and then dispatching a blur event on the top-level element in my custom component (the `div` element).

E.g:

```javascript
<template>
  <div>
     <input type="text" @blur="blur" />
  </div>
</template>

<script>
  export default {
    methods: {
      blur() {
        this.$el.dispatchEvent(new FocusEvent('blur'));
      },
    },
  }
</script>
```

**Comment by nathanreyes** [comment]
So the problem *seems* to go away if just handle the `focusout` event the same way I am for `blur`. Need to test if this doesn't introduce other problems, but if not then this should be a pretty easy fix.

**Comment by nathanreyes** [comment]
Ok I've tested this thing every which way I can with a preliminary fix in `1.0.0-beta.10`. Please advise if this still doesn't solve the issue.
